.babelrc

```diff
@@ -1,4 +1,5 @@
 {
   "presets":  ["es2015", "react"],
+  "plugins": ["transform-object-rest-spread"],
```

Comment:
`Object.assign( {}, { foo: 'bar' }, anotherObject } )` is equivalent to `{ foo: 'bar', ...anotherObject }`

There are some cases where it makes more sense to use the object spread syntax. In the reducers we can keep using Object.assign, or lodash/assign.

Personally, I think we should move to using lodash in the reducers. Lodash has a bunch of useful methods that transform objects but return clones: merge, assign, mergeWith, mergeIn, etc. With Object, we just have Object.assign.